src/_locales/en/messages.json

```diff
@@ -1,95 +1,96 @@
+    "dashboard_backup_persona_hint": "You can backup your persona in the following options.",
+    "dashboard_backup_persona_mnemonic_hint": "Keep the 12 mnemonic words above carefully in a safe place. You will need them to restore the private key of this persona.",
     "dashboard_backup_persona_qr_hint": "Keep the QR code carefully in a safe place. You will need them to restore the private key of this persona.",
     "dashboard_backup_persona_text_hint": "Keep the text above carefully in a safe place. You will need them to restore the private key of this persona.",
     "dashboard_bio_test_failed": "The profile bio should include {{bio}}. Please make sure you updated the profile bio of @{{name}} successfully.",
-    "dashboard_bio_test_succeeful": "You have successfully connected @{{profile}} on {{domain}} to persona {{persona}}.",
-    "dashboard_connect_profile_for": "Connect Profile for {{persona}}",
+    "dashboard_bio_test_succeeful": "You have successfully connected your persona {{persona}} to @{{profile}} on {{domain}}.",
+    "dashboard_connect_profile_for": "Connect {{persona}} to your profile",
     "dashboard_connected": "Connected",
     "dashboard_create_backup": "Backup",
     "dashboard_create_persona_hint": "Create a new persona.",
     "dashboard_database_import_successful_hint": "Your database has been restored. Existing data will be merged.",
     "dashboard_delete_persona": "Delete",
-    "dashboard_delete_persona_confirm_hint": "Do you want to delete persona {{name}}? This operation cannot be reverted.",
-    "dashboard_disconnect_profile_hint": "Do you want to disconnect {{profile}} on {{network}} from persona {{persona}}? This operation cannot be reverted.",
+    "dashboard_delete_persona_confirm_hint": "Do you want to delete the persona {{name}}? This operation cannot be reverted.",
+    "dashboard_disconnect_profile_hint": "Do you want to disconnect {{profile}} on {{network}} from the persona {{persona}}? This operation cannot be reverted.",
     "dashboard_download": "Download as File",
     "dashboard_export_keystore": "Export Backup Keystore",
     "dashboard_hide_qr": "Hide QR Code",
-    "dashboard_import_backup": "Import Data Backup",
-    "dashboard_import_database_hint": "From a previous database backup.",
+    "dashboard_import_backup": "Import a data backup",
+    "dashboard_import_database_hint": "Restore from a previous database backup.",
     "dashboard_import_persona_failed": "Your import data is invalid. Please check again.",
-    "dashboard_import_persona_hint": "From a previous persona backup.",
+    "dashboard_import_persona_hint": "Restore from a previous persona backup.",
     "dashboard_imported_persona": "Imported persona `{{name}}` with {{count}} profiles.",
-    "dashboard_init_header": "Post on social networks without allowing the corporations to stalk you.",
+    "dashboard_init_header": "Post on social networks without allowing the platforms to stalk you.",
```

Comment:
One minor note: We are technically using "platform" and "network" to refer to Chrome etc and Facebook etc. We should get a consensus on terminology, but I will leave this message as it is right now.